l10n_fi_invoice/models/account_move.py

```diff
@@ -18,21 +18,12 @@ class AccountMove(models.Model):
     def _get_invoice_pdf_filename(self):
         type_string = 'Invoice'
         invoice_numbers = self.name or ''
-        if self.type in ('in_refund', 'out_refund'):
+        if self.move_type in ('in_refund', 'out_refund'):
             type_string = 'Credit_Note'
         filename = '-'.join((
             type_string,
             invoice_numbers,
             self.company_id.display_name,
-            self.partner_id.display_name)). \
+            self.partner_id.display_name or '')). \
             replace(' ', '-').replace(',', '').replace('--', '-')
         return filename
-
-    def invoice_print(self):
```

Comment:
I think this is never called in 13.0. This was probably supposed to be `action_invoice_print` ([link to code](https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L2514)). I have removed it here as it doesn't seem to be used anyway. What do you think about it?

Reply:
Yeah, probably a leftover from older Odoo versions: in 10 it was still called just `invoice_print` https://github.com/odoo/odoo/blob/10.0/addons/account/models/account_invoice.py#L394

Seems fine to me. If you're migrating this for some specific project, you might want to ensure that the expected user flows still work. This seems to have been intended to make it so that when you press "Print & Send", it would use the finnish invoice PDF by default. It appears to have been broken in 13, but now we're removing it explicitly. So you might want to make sure the user flow is still as expected: it isn't too difficult to get this PDF printed when sending invoices?

Reply:
@aisopuro I've added your comment/explanation from that issue to a readme.

As for myself, it was a little bit of a struggle at first to get a barcode to print (as it depends on so many conditions, needed to look into code directly to follow). Do you think we should also list these conditions in `l10n_fi_invoice` readme (barcode func. comes from another module - `l10n_fi_bank_barcode`)?

It is added here though https://github.com/avoinsystems/avoinsystems-addons/blob/13.0/l10n_fi_bank_barcode/__manifest__.py#L28. Do you think that's enough?